### ISSUE TITLE
Dealing with overfitting

### ISSUE THREAD

**Issue by rimphyd** [tpfister/caffe-heatmap#16]
Hello all,
I've tried using caffe-heatmap with only small training data, which obviously end up as overfit model.
My main problem is that not only the training data is small, but it also have some redundancy. So, I thought I should leave cross-validation aside.
Now I'm trying to modify the euclidean_loss_heatmap_layer.cpp to see if I can rig the loss if certain condition is met.

Has anyone ever experience overfitting using caffe-heatmap? I'll be gratefull to hear some pointers.

Thank you.
